tests/python/core/test_multi_function_flow.py

```diff
@@ -0,0 +1,249 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=protected-access, attribute-defined-outside-init, too-many-locals
+# pylint: disable=too-many-statements, no-self-use, too-many-arguments
+import raf
+from raf._core.ndarray import get_ndarray_handle
+from raf.ir import ScopeBuilder
+from raf.model.trace import _get_func_inputs
+from raf._core.device import Device
+from raf._core.executor import VMExecutor
+from raf._ffi.model import RunModel
+from raf._ffi.memory_pool import InitPool
+from raf.testing import check, randn
+from raf.model.trace import _unwrap
+
+import tvm
+from tvm import relay
+
+# Run a module through the VM, return the result in numpy
+def run_vm(model_or_mod, args, use_multi_func=False):
+
+    if not isinstance(model_or_mod, tvm.IRModule):
+        record = model_or_mod._internal(*args)
+        args = _get_func_inputs(record, args, {}, get_handle=False)
+        mod = record.mod
+    else:
+        mod = model_or_mod
+
+    # Use CPU to avoid workspace memory.
+    device = "cpu"
+    InitPool(Device(device), "page_unit_pool")
+    pass_config = {"raf.memory_schedule": True, "raf.memory_budget": int(13e9)}
```

Comment:
Why you need to avoid workspace memory in these tests? And do you need to enable memory passes?

Reply:
That was a mistake. We don't need to avoid workspace memory and I will change it in the next revision. 

Testing whether the memory passes would work after inlining is actually the purpose of this test, so we need to enable them.